cmd/amppkg/main.go

```diff
@@ -91,7 +91,7 @@ func main() {
 	if certs == nil || len(certs) == 0 {
 		die(fmt.Sprintf("no cert found in %s", config.CertFile))
 	}
-	if (!*flagDevelopment && !*flagInvalidCert) && !util.CanSignHttpExchanges(certs[0]) {
+	if !(*flagDevelopment || *flagInvalidCert) && !util.CanSignHttpExchanges(certs[0]) {
```

Comment:
Oops, actually it reads even better if you De Morgan again:
`!(*flagDevelopment || *flagInvalidCert || util.CanSignHttpExchanges(certs[0]))`

But that's outside the scope of your change, so feel free to ignore.

Reply:
Resolved in commit 79453d9